distro/configuration/metadatatermmappings/metadatatermmappings.csv

```diff
@@ -1,3 +0,0 @@
-Uuid,Void/Retire,Mapping code,Mapping source,Metadata class name,Metadata Uuid,_order:2000
```

Comment:
For a while these don't seem to be necessarily needed (since no one is noticing their absence in a Ref App distro) and so IMO, removing them won't do any harm I guess.

Reply:
@ibacher, I've been looking into this silent failure to load the `emrapi.Metadata_Source-1.zip` and the PR below should fix it.

https://github.com/mekomsolutions/openmrs-module-initializer/pull/241